tests/MassTransit.Metrics.Tests/Metrics_Specs.cs

```diff
@@ -0,0 +1,95 @@
+namespace MassTransit.Metrics.Tests
+{
+    using MassTransit.Metrics.Tests.HarnessContracts;
+    using MassTransit.Monitoring;
+    using MassTransit.Testing;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Diagnostics.Metrics.Testing;
+    using NUnit.Framework;
+    using System.Threading.Tasks;
+
+    namespace HarnessContracts
+    {
+        using System;
+
+        public interface SubmitOrder
+        {
+            Guid OrderId { get; }
+            string OrderNumber { get; }
+        }
+
+
+        public interface OrderSubmitted
+        {
+            Guid OrderId { get; }
+            string OrderNumber { get; }
+        }
+    }
+
+
+    [TestFixture]
+    [Explicit]
+    public class Metrics_Specs
+    {
+        [Test]
+        public async Task Should_get_metrics_with_tags()
+        {
+            var collector = new MetricCollector<long>(null, InstrumentationOptions.MeterName, "messaging.masstransit.consume");
+
+            var services = new ServiceCollection();
+
+            await using var provider = services
+                .AddMassTransitTestHarness(x =>
+                {
+                    x.AddConsumer<MonitoredSubmitOrderConsumer>();
+
+                    x.UsingActiveMq((context, cfg) =>
```

Comment:
not ActiveMq please, why not just InMemory? or otherwise rmq

Reply:
I wanted to use the InMemory, but the logging methods were not triggered for some reason. In a few days, I can try it again. I am afk.

Reply:
> I'd rename them to OpenTelemetry.Tests (something more generic)
I renamed the test project to MassTransit.Observability.Tests.

> not ActiveMq please, why not just InMemory? or otherwise rmq
`InMemory` transport didn't work for me. I switched to rmq.